test/python/sql_validation.py

```diff
@@ -237,7 +237,7 @@ def list_upgrade_files():
     inpPath = "../JDBC/upgrade"
 
     # searching prepare and verify scripts in upgrade directory
-    update_files = list_files(inpPath, "**/*[-][v][u][-][!c]*.*")
+    update_files = list_files(inpPath, "**/*[-][v][u][-]*.*")
```

Comment:
why can't we add drop server testing to verify scripts? cleanup script is meant to cleanup test objects but if testing itself involves dropping objects, it should be done in prepare/verify